distribution/docs/src/main/resources/_contents/_data/data-intro-contents.adoc

```diff
@@ -0,0 +1,359 @@
+
+==== Metacard Type
+
+A metacard type indicates the attributes available for a particular metacard.
+It is a model used to define the attributes of a metacard, much like a schema.
+
+===== Default Metacard Type and Attributes
+
+Most metacards within the system are created using with the default metacard type.
+The default metacard type of the system can be programmatically retrieved by calling `${ddf-branding}.catalog.data.BasicTypes.BASIC_METACARD`.
+The name of the default MetacardType can be retrieved from `${ddf-branding}.catalog.data.MetacardType.DEFAULT_METACARD_TYPE_NAME`.
+
+The default metacard type has the following required attributes.
+Though the following attributes are required on all metacard types, setting their values is optional except for ID.
+
+.[[_metacard_required_attributes]]Metacard Required Attributes
+[cols="4*", options="header"]
+|===
+
+|`${ddf-branding}.catalog.data.Metacard` Constant
+|Attribute Name
+|Attribute Format
+|Description
+
+|`CONTENT_TYPE`
+|`metadata-content-type`
+|STRING
+|Attribute name accessing for the metadata content type of a Metacard.
+
+|`CONTENT_TYPE_VERSION`
+|`metadata-content-type-version`
+|STRING
+|Attribute name for the version of the metadata content accessing type of a Metacard.
+
+|`CREATED`
+|`created`
+|DATE
+|Attribute name for accessing the date/time *this Metacard* was created.
+
+|`EFFECTIVE`
+|`effective`
+|DATE
+|Attribute name for accessing the date/time of *the product* represented by the Metacard.
+
+|`EXPIRATION`
+|`expiration`
+|DATE
+|Attribute name for accessing the date/time the Metacard is no longer valid and could be removed.
+
+|`GEOGRAPHY`
+|`location`
+|GEOMETRY
+|Attribute name for accessing the location for this Metacard.
+
+|`ID`
+|`id`
+|STRING
+|Attribute name for accessing the ID of the Metacard.
+
+|`METADATA`
+|`metadata`
+|XML
+|Attribute name for accessing the XML metadata for this Metacard.
+
+|`MODIFIED`
+|`modified`
+|DATE
+|Attribute name for accessing the date/time this Metacard was last modified.
+
+|`RESOURCE_SIZE`
+|`resource-size`
+|STRING
+|Attribute name for accessing the size in bytes of the product this Metacard represents.
+
+|`RESOURCE_URI`
+|`resource-uri`
+|STRING
+|Attribute name for accessing the URI reference to the product this Metacard represents.
+
+|`TARGET_NAMESPACE`
+|`metadata-target-namespace`
+|STRING
+|Attribute name for the target namespace of the accessing metadata content type of a Metacard.
+
+|`THUMBNAIL`
+|`thumbnail`
+|BINARY
+|Attribute name for accessing the thumbnail image of the product this Metacard represents. The thumbnail must be of MIME Type `image/jpeg` and be less than 128 kilobytes. 
+
+|`TITLE`
+|`title`
+|STRING
+|Attribute name for accessing the title of the Metacard.
+
+|===
+
+[NOTE]
+====
+It is highly recommended when referencing a default attribute name to use the `ddf.catalog.data.Metacard` constants whenever possible.
```

Comment:
`ddf.catalog.data.Metacard` is probably going to be deprecated in 2.10 in favor of `ddf.catalog.data.types.Core`.